examples/AddData.R

```diff
@@ -0,0 +1,51 @@
+###
+# Add dataframes, CSV files, plots, etc. to OMERO
+###
+
+library(romero.gateway)
+
+# Load "Iris" example dataset
+library(datasets)
+data("iris")
+data.frame(iris)
+
+# Connect to server
+server <- OMEROServer(host = "localhost", username = "xxx", password = "xxx")
+server <- connect(server)
+
+# Load the image (can be any other OME object, too) we want to attach the
+# data to, in that case the image with ID=1
+imageId <- 1
+image <- loadObject(server, "ImageData", imageId)
+
+# Attach the dataframe to the image.
+# Stores the data as OMERO HDF5 table, see
+# https://docs.openmicroscopy.org/omero/5.4.7/developers/Tables.html
```

Comment:
https://docs.openmicroscopy.org/latest/omero/developers/Tables.html unless you want to pin to a specific version